src/templates/Home/Column.tsx

```diff
@@ -25,7 +27,8 @@ export const Column: React.FC<ColumnProps> = ({
   colId,
   onOpenCreateTodoModal,
 }) => {
-  const todoList = useZustand((store) => store.todos[colId]);
+  const todoList = useZustand((store) => store.todos);
```

Comment:
Querying the the store this way causes all the columns to update when a single task changes.

Reply:
Fixed